opus/application/static_media/js/browse.js

```diff
@@ -1947,12 +1956,29 @@ var o_browse = {
         o_browse.metadataboxHtml(opusId);
     },
 
-    clearObservationData: function() {
-        opus.prefs.startobs = 1; // reset startobs to 1 when data is flushed
-        opus.prefs.cart_startobs = 1;
+    clearObservationData: function(leaveStartObs) {
+        if (!leaveStartObs) {
+            // Normally when we delete all the data we want to force a return to the top because
+            // we don't know what data is going to be loaded. But in some circumstances
+            // (like updating metadata columns) we know we're going to reload exactly the same
+            // data so we can keep our place.
+            opus.prefs.startobs = 1; // reset startobs to 1 when data is flushed
+            opus.prefs.cart_startobs = 1;
+        }
         o_cart.reloadObservationData = true;  // forces redraw of cart tab
         o_cart.observationData = {};
         o_browse.reloadObservationData = true;  // forces redraw of browse tab
         o_browse.observationData = {};
     },
+
+    clearBrowseObservationDataAndEraseDOM: function(leaveStartObs=false) {
```

Comment:
why don't you need to do this on the cart for similar reasons as well....?

Reply:
clearBrowseObservationDataAndEraseDOM is called from the main OPUS UI when the user changes the search parameters. It makes sure that a data load will be triggered in the browse tab, even if the user switches tabs at an inopportune time (like we didn't actually send out for a result_count yet). But the cart isn't affect by the user's search, so there's no point in clearing it. Since there are now two separate caches for data, there's no overlap between the tabs.